04/app.js

```diff
@@ -6,4 +6,23 @@ const user = {
         month: '04',
         year: '1985'
     }
-}
\ No newline at end of file
+}
+
+const currDate = (new Date).toLocaleDateString();
+
+user.checkBirthday = function(date) {
+
+    if (typeof this.born === 'undefined') {
+        return 'Nie podano danych urodzenia';
+    }
+
+    const userBornDate = this.born.day + '.' + this.born.month + '.' + this.born.year;
```

Comment:
oj oj - czy sprawdziłeś czy to faktycznie działa?

Przy określaniu urodzin potrzebujemy tylko dnia i miesiąca, a nie całego roku - rodzimy się tylko raz, ale wielokrotnie obchodzimy rocznicę, w której rok nie jest tak istotny (przynajmniej dla mężczyzn) :P